File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        for(int i=0;i<s.size();i++){
            if((s[i]>=65&&s[i]<=90)||(s[i]>=97&&s[i]<=122)||(s[i]>=48&&s[i]<=57)){
                char c = tolower(s[i]); 
                s[i]=c;
                continue;
            } 
            else{
                s.erase(i,1);
                --i;
            }
        } 
        cout<<s<<endl;
        int i=0; 
        int j=s.size()-1; 
        while(i<j){
            if(s[i]!=s[j]){
                return false;
            } 
            else{
                i++;
                j--;
            }
        }
        return true;
    }
};

File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        s = s.toLowerCase();
        String revstr1 = "";
        int i ;
        int strl = s.length();
        for (i = (strl-1); i >= 0 ; i--)
        {   
            char ch=s.charAt(i);
            if((ch>=65&&ch<=90)||(ch>=97&&ch<=122)||(ch>=48&&ch<=57)){
                revstr1 = revstr1 + s.charAt(i);
            }
            
        } 
        String revstr2 = "";
        for (i =0; i<strl ; i++)
        {   
            char ch=s.charAt(i);
            if((ch>=65&&ch<=90)||(ch>=97&&ch<=122)||(ch>=48&&ch<=57)){
                revstr2 = revstr2 + s.charAt(i);
            }
            
        }
        System.out.println(revstr1);
if (revstr2.equals(revstr1)){
return true;}
else 
return false;}}